### PR TITLE
Update the version number

### PR DESCRIPTION
**Pull request by kdorheim**
Update the version number to v3.2! I believe I got all the places that state V3.1.1,  @bpbond will you give it a quick once over?

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# hector 3.2.0 
+
+* Correct aerosol forcing coefficients based on Zelinka et al. (2023)
+* Enable permafrost module
```
Comment (bpbond) [inline NEWS.md:4]: Perhaps
```
* Enable permafrost module and recalibrate model's default parameterization
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ efficiency, SCMs can easily be coupled to other models and used to
 design scenarios, emulate more complex climate models, and conduct
 uncertainty analyses.
 
+    #> Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
+    #> ℹ Please use `linewidth` instead.
+    #> This warning is displayed once every 8 hours.
+    #> Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
+    #> generated.
+
```
Comment (bpbond) [inline README.md:29]: We don't want to commit this--please remove! It looks like this is coming from line 39 in the Rmd file, so change that to
```
    geom_line(aes(year, value, color = scenario), linewidth= 1) +
```

**Review by bpbond — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kdorheim) [reply README.md:29]: nice catch! Thanks!

**Review by bpbond — APPROVED** [review]
Looks good! Thanks @kdorheim

**Pull request merged**